p2p/transport/webtransport/multiaddr.go

```diff
@@ -78,3 +75,59 @@ func addrComponentForCert(hash []byte) (ma.Multiaddr, error) {
 	}
 	return ma.NewComponent(ma.ProtocolWithCode(ma.P_CERTHASH).Name, certStr)
 }
+
+// IsWebtransportMultiaddrWithCerthash returns true if the given multiaddr is a
+// well formed webtransport multiaddr. Returns the number of certhashes found.
+func IsWebtransportMultiaddr(multiaddr ma.Multiaddr) (bool, int) {
+	const (
+		init              = iota
+		foundUDP          = iota
+		foundQuicV1       = iota
+		foundWebTransport = iota
```

Comment:
One `iota` should be enough. I don't know how multiple of them behave, does it reset to 0?

Reply:
I was following the [docs](https://go.dev/ref/spec#Iota). It's the same thing.